Dockerfile

```diff
@@ -147,8 +147,13 @@ RUN git clone https://github.com/genome/bam-readcount.git /tmp/bam-readcount-0.7
 ##########
 #fpfilter#
 ##########
+WORKDIR /opt
+wget --no-check-certificate https://raw.githubusercontent.com/genome/fpfilter-tool/v0.1.0/fpfilter.pl
 COPY fpfilter.pl /usr/bin/fpfilter.pl
```

Comment:
Since the file is no longer in the repo, this will need to be replaced by a `cp` or `mv`.